### ISSUE TITLE
Training Loss Curve

### ISSUE THREAD

**Issue by OctopusDOC** [marrlab/DinoBloom#7]
Hi Marr Lab,

First of all, congratulations on the impressive model you developed!

I am also using the dinov2 framework and the pre-trained weights it provides to continue fine-tuning on MRI images, or more accurately, this is similar to Continued Pretraining(CPT), which is quite similar to you work.

I encountered some problems during training, and I hope to communicate with you to gain some experience: 
I found that when I trained on a dataset(~22k), the training loss only dropped from 14.09 to 13.6 (I did not remove the dino loss), and then stopped dropping. Is this abnormal?

If you can provide some details of the training loss, or a complete training loss curve, I will be very grateful :)

**Comment by ValentinKoch** [comment]
Hey, thanks for your interest! Thats super hard to diagnose with so little details. In this work we provide some curves: https://arxiv.org/pdf/2401.04720. Probably your loss should get lower, did you check if all settings are correct like unfreezing weights etc? Otherwise maybe your batch size is too small? But these are just wild guesses. Best of luck!
